packages/components/psammead-most-read/src/list.jsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import styled from 'styled-components';
+import { shape, string, oneOf, arrayOf } from 'prop-types';
+import {
+  Burmese,
+  Bengali,
+  EasternArabic,
+  WesternArabic,
+} from '@bbc/psammead-locales/numerals';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import Grid from '../../psammead-grid';
+import { MostReadRank, MostReadLink } from './item';
+
+const MostReadItemProps = {
+  columns: {
+    group0: 18,
+    group1: 18,
+    group2: 9,
+    group3: 9,
+    group4: 12,
+    group5: 12,
+  },
+};
+
+const MostReadRankProps = {
+  item: true,
+  columns: {
+    group0: 2,
+    group1: 2,
+    group2: 2,
+    group3: 2,
+    group4: 2,
+    group5: 2,
+  },
+};
+
+const MostReadLinkProps = {
+  item: true,
+  columns: {
+    group0: 16,
+    group1: 16,
+    group2: 7,
+    group3: 7,
+    group4: 10,
+    group5: 10,
+  },
+};
+
+const MostReadListProps = {
+  enableGelGutters: true,
+  enableGelMargins: true,
+  columns: {
+    group0: 18,
```

Comment:
These numbers have been adjusted to be three times the default base number of columns (6,6,6,6,8,20) originally to control the spacing between `rank` and `link` more easily.